### ISSUE TITLE
`@stylistic/function-parentheses-space-inside` fix takes the code behind a function into an inline comment

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#280]
Found while fixing #132, by checking whether the rewriting that issue asks for belongs on the opening parenthesis as well. It does not belong here, because there is nothing to rewrite: this rule carries no guard on its opening parenthesis at all. The closing one has been guarded since #114; the opening one breaks a file through the same door #129 named in the sibling rule, and no issue names it here.

### What minimal example or steps are needed to reproduce the bug?

```less
a {
  t: foo(1px // c) calc(
2px);
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-less",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-space-inside": "never"
  }
}
```

### What did you expect to happen?

The line break in front of `2px` is what closes the `// c) calc(` comment, so `never` cannot take it away without commenting out everything behind it. The fix should leave the value alone and let the warning stand, as it does for the closing parenthesis.

### What actually happened?

```less
a {
  t: foo(1px // c) calc(2px);
}
```

Everything from the double slash to the end of the line is comment text now, the declaration ends nowhere, and no warning survives the run: `--fix` reports a clean pass on a file it has just broken.

The cause is at [`lib/rules/function-parentheses-space-inside/index.js:133-163`](https://github.com/stylelint-stylistic/stylelint-stylistic/blob/main/lib/rules/function-parentheses-space-inside/index.js#L133-L163), where all four options write `valueNode.before` without asking what stands in front of that whitespace. Here that whitespace is `calc`'s own `before`, and what stands in front of it is the tail of a comment opened inside `foo`.

### How much is at stake

The same arithmetic #129 wrote about itself: for the input to exist, the comment must already swallow a closing parenthesis of its own, so the file is already unbalanced for less.js and for Sass alike, and only `postcss-less` reads it at all — it keeps a double slash in a value as ordinary text, which is what #32 was about. No valid stylesheet reaches this. What makes it worth an issue rather than a note is the same thing that made #129 one: the sweep #113, #114, #116 and #117 belong to exists to leave no fixer writing into a comment.

### Do you have a proposal to fix the bug?

The shape the cluster settled on, mirrored from #129's repair of the sibling rule: where the text the fix would leave in front of the first argument ends with an inline comment, report the problem and write nothing. `endsWithInlineComment` in `lib/utils/` answers that. Both askings should be put to each reading of a line break the way #132 leaves the closing guard doing it, and #281 asks the sibling rule's opening guard for the same.
